internal/signer/darwin/keychain/keychain_test.go

```diff
@@ -46,3 +50,147 @@ func TestBytesToCFDataRoundTrip(t *testing.T) {
 		t.Errorf("bytesToCFData -> cfDataToBytes\ngot  %x\nwant %x", got, want)
 	}
 }
+
+func TestEncryptRSA(t *testing.T) {
+	hashFunc := sha256.New()
+	rng := rand.Reader
+	key, errCred := Cred("enterprise_v1_corp_client-signer-0-2018-07-03T10:55:10-07:00 K:1, 2:BXmhnePmGN4:0:18")
```

Comment:
Is there a test credential being used in any other macos tests? Eventually we will need to see if we can create one in the CI, because this assumes we have the certificate provisioned to the device.

Reply:
Hmm I don't think there are any test credentials being used so I'm thinking we will need to create one in the CI

Reply:
i think there are some bash commands for the `security` command group that we can use in the CI.

Don't worry about it for now, unless you have extra time